### ISSUE TITLE
Editor: Load map, dropdown "Display: Map Names" should be radiogroup

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#6377]
<img align='left' src='https://codeberg.org/avatars/94062fb4c3e503b6434a88938aca6486630fcdfe4d13f557dc2ef771ba3943d8' alt='frankystone' height='48' />*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/4740)*
*Created on Mon Feb 26 20:12:03 CET 2024 by **frankystone***

---

If you want to save a map just with another name (without changing it's name in the map options) you can do that by entering text in the filename editbox.

But if you want to load this map you always have to click twice on the dropdown "Display: ...": One time for opening the dropdown and one time for choosing "File Names".

**Describe the solution you'd like**
The dropdown should be replaced with a radiogroup. "Map Names" should be the default like in the dropdown. But if a user selected "File Names" this should be saved to the config so it becomes the new default.

**Describe alternatives you've considered**
None

**Additional context**
As far i can see this dropdown (or option) is only used for loading maps in the editor, so it should not interfere with the game's load map menu. 

**Implementation steps**
- [ ] ...
- [ ] ...
- [ ] ...

**Comment by bunnybot** [comment]
<img align='left' src='https://codeberg.org/avatars/94062fb4c3e503b6434a88938aca6486630fcdfe4d13f557dc2ef771ba3943d8' alt='frankystone' height='48' />*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/4740#issuecomment-1589803)*
*On Mon Feb 26 20:22:22 CET 2024, **frankystone** wrote:*

---

As a first step saving the chosen entry of the dropdown would be good. Adding textual radio buttons can be implemented later, imho.
